apps/site/lib/site/icalendar_generator.ex

```diff
@@ -6,38 +6,41 @@ defmodule Site.IcalendarGenerator do
   @spec to_ical(Plug.Conn.t(), Event.t()) :: iodata
   def to_ical(%Plug.Conn{} = conn, %Event{} = event) do
     [
-      "BEGIN:VCALENDAR\n",
-      "VERSION:2.0\n",
-      "PRODID:-//www.mbta.com//Events//EN\n",
-      "BEGIN:VEVENT\n",
+      "BEGIN:VCALENDAR\r\n",
+      "VERSION:2.0\r\n",
+      "PRODID:-//www.mbta.com//Events//EN\r\n",
+      "BEGIN:VEVENT\r\n",
       "UID:",
       "event",
       "#{event.id}",
       "@mbta.com",
-      "\n",
+      "\r\n",
       "SEQUENCE:",
+      unix_time(),
+      "\r\n",
+      "DTSTAMP:",
       timestamp(),
-      "\n",
+      "\r\n",
       "DTSTART:",
       start_time(event),
-      "\n",
+      "\r\n",
       "DTEND:",
       end_time(event),
-      "\n",
+      "\r\n",
       "DESCRIPTION:",
       description(event),
-      "\n",
+      "\r\n",
       "LOCATION:",
       address(event),
-      "\n",
+      "\r\n",
       "SUMMARY:",
       event_summary(event),
-      "\n",
+      "\r\n",
       "URL:",
       full_url(conn, event),
-      "\n",
-      "END:VEVENT\n",
-      "END:VCALENDAR\n"
+      "\r\n",
+      "END:VEVENT\r\n",
+      "END:VCALENDAR\r\n"
     ]
```

Comment:
You could also use your new function `replace_newlines` here instead of adding the `\r`:

```elixir
[...]|>Enum.map(fn str -> replace_newlines(str) end)
```